### PR TITLE
Feat(eos_cli_config_gen): logging facility

### PR DESCRIPTION
**Pull request by carlbuchmann**
## Change Summary

Add support to define logging facility

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes

```yaml
logging:
  facility: < syslog_facility_value >
```

## How to test

See Molecule test case.

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by tgodaA — APPROVED** [review]
LGTM

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md [hunk 1]
```diff
@@ -2024,6 +2024,7 @@ logging:
     timestamp: < high-resolution | traditional >
     hostname: < fqdn | ipv4 >
     sequence_numbers: < true | false >
+  facility: < syslog facility value >
```
Comment (tgodaA) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md]: I am not sure but some people might get confused, if this is a single value or multiple. So please rephrase this to ```syslog_facility_value```

**Review by emilarista — APPROVED** [review]
LGTM!
